spec/PostgresStorageAdapter.spec.js

```diff
@@ -19,4 +30,29 @@ describe_only_db('postgres')('PostgresStorageAdapter', () => {
     expect(adapter._client.$pool.ending).toEqual(true);
     done();
   });
+
+  it('schemaUpgrade, upgrade the database schema when schema change', done => {
+    const adapter = new PostgresStorageAdapter({ uri: databaseURI });
+    const client = adapter._client;
+    const className = '_PushStatus';
+    const schema = {
+      fields: {
+        "pushTime": { type: 'String' },
+        "source": { type: 'String' }, // rest or webui
+        "query": { type: 'String' }, // the stringified JSON query
+      },
+    };
+
+    adapter.createTable(className, schema)
+      .then(() => {
+        schema.fields.expiration_interval = { type:'Number' };
```

Comment:
Just to be pedantic can we** getColumns** and check before the schema is upgraded to ensure that it's not present before hand?